packages/intrepid2/src/Projection/Intrepid2_LagrangianInterpolationDef.hpp

```diff
@@ -56,6 +56,217 @@
 namespace Intrepid2 {
 namespace Experimental {
 
+
+
+template<typename scalarViewType,
+typename ortViewType,
+typename t2oViewType,
+typename subcellParamViewType,
+typename intViewType>
+struct F_computeDofCoordsAdCoeff {
```

Comment:
Typo?  Should be `F_computeDofCoordsAndCoeff`,  I think…

Reply:
oh, right, I'll fix it